project-backend/Models/Validators/BidController/AddBid/AddBidQueryValidator.cs

```diff
@@ -0,0 +1,18 @@
+﻿using FluentValidation;
+using project_backend.Models.BidController.AddBid;
+
+namespace project_backend.Models.Validators.BidController.AddBid
+{
+    public class AddBidQueryValidator : AbstractValidator<AddBidQueryObject>
+    {
+        public AddBidQueryValidator()
+        {
+            const string negativeSumError = "Sum must be positive";
+            const string sumMissingError = "Sum must not be null";
+            const string jobIdMissingError = "Job id must not be null";
+
+            RuleFor(x => x.Sum).NotNull().WithMessage(sumMissingError).GreaterThanOrEqualTo(0).WithMessage(negativeSumError);
```

Comment:
I have no clue if we should make this stricter (> 0) or we can leave it as >= 0

Reply:
What if someone wants to do it for free? :))